### PR TITLE
Reviews

### PR DESCRIPTION
**Pull request by Lomig**
# Reviews M.C

- View rendered in Bars#show _(untested)_
- New/Create rendered in Bookings#show **(not yet implemented)**

### CODE REVIEW

File: app/controllers/bars_controller.rb [hunk 1]
```diff
@@ -16,7 +16,9 @@ def index
     end
   end
 
-  def show; end
+  def show
+    @review = Review.new
+  end
```
Comment (dornby) [inline app/controllers/bars_controller.rb]: Toujours d'actualité ca ?
Vu que tu l'as migré vers Booking

Comment (Lomig) [reply app/controllers/bars_controller.rb]: Nope

File: app/policies/bar_policy.rb [hunk 2]
```diff
@@ -10,11 +10,11 @@ def show?
   end
 
   def create?
-    true
+    record.user == user
   end
 
   def update?
-    record.owner == user
+    true
```
Comment (dornby) [inline app/policies/bar_policy.rb]: Hmmm, anybody could update any review?

Comment (dornby) [inline app/policies/bar_policy.rb]: Hmm, t'es sûr de vouloir changer ca ?
Anybody should create
Only owners should update

**Pull request merged**